examples/spark/images/base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:latest
+FROM java:openjdk-8-jdk
```

Comment:
`Building Spark using Maven requires Maven 3.3.3 or newer and Java 7+`

But I was actually pegging it because I didn't want the Zeppelin build process to go all over the process (in the mythical world where an `openjdk-9-jdk` gets released).